fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -4077,12 +4077,30 @@ public ParseNode visitRowConstructor(StarRocksParser.RowConstructorContext conte
 
     @Override
     public ParseNode visitPartitionNames(StarRocksParser.PartitionNamesContext context) {
+        if (context.lakePartition() != null) {
+            return visit(context.lakePartition());
+        }
+
         List<Identifier> identifierList = visit(context.identifier(), Identifier.class);
         return new PartitionNames(context.TEMPORARY() != null,
                 identifierList.stream().map(Identifier::getValue).collect(toList()),
                 createPos(context));
     }
 
+    @Override
+    public ParseNode visitLakePartitions(StarRocksParser.LakePartitionsContext context) {
+        List<String> partitionNames = Lists.newArrayList();
+        List<Expr> partitionValues = Lists.newArrayList();
+        for (StarRocksParser.PartitionPairContext pair : context.partitionPair()) {
+            Identifier partitionName = (Identifier) visit(pair.key);
```

Comment:
partition name in internal table is not the partition column name, could you please refer to this: https://docs.starrocks.io/en-us/3.0/sql-reference/sql-statements/data-definition/CREATE%20TABLE#partition_desc